.github/workflows/golangci-lint.yaml

```
@@ -14,19 +14,19 @@ jobs:
     runs-on: ubicloud-standard-4
     steps:
       - name: Check out code
-        uses: actions/checkout@v5
+        uses: actions/checkout@v7
 
       - name: Set Go Version
         id: go-version
         run: echo "version=$(sed -En 's/^go (.*)$/\1/p' go.mod)" >> $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:56: Double quote to prevent globbing and word splitting

(shellcheck)